Provis.Core/Services/WorkspaceService.cs

```diff
@@ -296,5 +296,34 @@ public async Task<List<WorkspaceMemberDTO>> GetWorkspaceMembersAsync(int workspa
 
             return workspaceMembers;
         }
+
+        public async Task CancelInviteAsync(int id, int workspaceId, string userId)
+        {
+            var invite = await _inviteUserRepository
+                .Query()
+                .FirstOrDefaultAsync(i => i.Id == id &&
```

Comment:
Please compare execution plan for selected query with just Id and getByKeyASync method and your realization

Reply:
I tried to compare and as a result, there was almost no difference between them but "GetByKey" is a better way to get value.